opentelemetry-sdk/tests/performance/resource-usage/trace/profile_resource_usage_batch_export.py

```diff
@@ -1,49 +0,0 @@
-# Copyright The OpenTelemetry Authors
```

Comment:
Why these two files gets deleted and not moved?

Reply:
These files are not used anywhere. They are no longer relevant after [this](https://github.com/open-telemetry/opentelemetry-python/pull/3188/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5L99) was removed.